sdk/go/core/docsgen/api/types/ydk_types.rst

```diff
@@ -268,44 +268,172 @@ These are how YANG types are represented in Go.
 
     An interface type that represents a basic container in YANG
 
-    .. function:: GetEntityData()
+    .. function:: GetEntityData() Entity
+    
+        The :ref:`Entity <types-entity>` interface function
```

Comment:
This is vague. Instead say something like
Implements :ref:\`Entity <types-entity>\` interface

Reply:
Fixed.

Reply:
I did not see the change in your latest push - bumping this comment so it doesn't get buried